pennylane/measurements/measurements.py

```diff
@@ -269,7 +281,8 @@ def __init__(
     @property
     def return_type(self) -> Optional[ObservableReturnTypes]:
         """Measurement return type."""
-        return None
+        warn_return_type_deprecation()
+        return self._shortname
```

Comment:
So `MeasurementProcess.return_type` should still be accessible as an `ObservableReturnType` for the next cycle. It will just warn whenever it is accessed.